apps/meteor/server/models/raw/FreeSwitchChannel.ts

```diff
@@ -0,0 +1,67 @@
+import type { IFreeSwitchChannel, IFreeSwitchChannelEvent, RocketChatRecordDeleted } from '@rocket.chat/core-typings';
+import type { IFreeSwitchChannelModel } from '@rocket.chat/model-typings';
+import type { IndexDescription, Collection, Db, FindOptions, FindCursor } from 'mongodb';
+
+import { BaseRaw } from './BaseRaw';
+
+export class FreeSwitchChannelRaw extends BaseRaw<IFreeSwitchChannel> implements IFreeSwitchChannelModel {
+	constructor(db: Db, trash?: Collection<RocketChatRecordDeleted<IFreeSwitchChannel>>) {
+		super(db, 'freeswitch_channels', trash);
+	}
+
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{ key: { uniqueId: 1 }, unique: true },
+			{ key: { otherLegUniqueId: 1 }, unique: false },
+			{ key: { callId: 1 }, unique: false },
```

Comment:
from the query patterns, looks like this can be a single index:
```suggestion
			{ key: { uniqueId: 1, otherLegUniqueId: 1, callId: 1 }, unique: true },
```